src/merge_sort.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+
+def merge_sort(a_list):
+    """Use merge sort to sort the list."""
+    if len(a_list) <= 1:
+        return a_list
+    left = []
+    right = []
+    for i in range(len(a_list)):
```

Comment:
This is an unconventional way of splitting a list in half, and I'm loving it. Nice job.